zebra-instructions.md

```diff
@@ -91,72 +97,151 @@ Install dependencies:
     sudo apt-get install redis
     redis-server
     ```
+
 2. Install and activate a node version manager (e.g. [`nodenv`](https://github.com/nodenv/nodenv#installation) or [`nvm`](https://github.com/nvm-sh/nvm#installing-and-updating))
 3. Install `boost` and `libsodium` development libraries
-   On Ubuntu:
+
     ```sh
     sudo apt install libboost-all-dev
     sudo apt install libsodium-dev
     ```
 
-Install `s-nomp`:
+#### Install `s-nomp`
+
 1. `git clone https://github.com/ZcashFoundation/s-nomp`
 2. `cd s-nomp`
 3. Use the Zebra fixes: `git checkout zebra-mining`
-4. Use node 8.17.0:
+4. Use node 8.11.0:
+
     ```sh
-    nodenv install 8.17.0
-    nodenv local 8.17.0
+    nodenv install 8.11.0
+    nodenv local 8.11.0
     ```
-    or 
+
+    or
+
     ```sh
-    nvm install 8.17.0
-    nvm use 8.17.0
+    nvm install 8.11.0
+    nvm use 8.11.0
     ```
 
 5. Update dependencies and install:
+
     ```sh
     export CXXFLAGS="-std=gnu++17"
     npm update
     npm install
     ```
 
-Run `s-nomp`:
+</details>
+
+<details><summary>Arch-specific instructions</summary>
+    
+#### Install dependencies
+
+1. Install [`redis`](https://redis.io/docs/getting-started/) and run it on the default port:
+
+    ```sh
+    sudo pacman -S redis
+    sudo systemctl start redis
+    ```
+
+2. Install and activate [`nvm`](https://github.com/nvm-sh/nvm#installing-and-updating):
+
+    ```sh
+    sudo pacman -S nvm
+    unset npm_config_prefix
+    source /usr/share/nvm/init-nvm.sh
+    ```
+
+3. Install `boost` and `libsodium` development libraries:
+
+    ```sh
+    sudo pacman -S boost libsodium
+    ```
+
+#### Install `s-nomp`
+
+1. `git clone https://github.com/ZcashFoundation/s-nomp && cd s-nomp`
+
+2. Use the Zebra configs: `git checkout zebra-mining`
+
+3. Use node 8.11.0:
+
+    ```sh
+    nvm install 8.11.0
+    nvm use 8.11.0
+    ```
+
+4. Update dependencies and install:
+
+    ```sh
+    npm update
+    npm install
+    ```
```

Comment:
Optional: These instructions are exactly the same for Debian, Ubuntu, Arch, and most Linux, macOS, BSD.

Can we just list them once after the platform specific sections?

Reply:
I was trying to do this, but the general section mentions `nodenv`, which I tried, and it didn't work for me. It seems that `nvm` has better support on Arch, and it worked right away, so I omitted `nodenv`. I didn't find a suitable way to merge the instructions because of `nodenv`.

Reply:
Should I drop `nodenv`, and keep only `nvm`? It would shorten the instructions, and allow easy merging of the split parts.

Reply:
Or I could keep `nodenv` as an alternative, and put it into details. I'm going to try that.

Reply:
Thanks, I hadn't noticed the difference there. Happy to standardise on `nvm` if `nodenv` has compatibility issues.